taqo_client/lib/main.dart

```diff
@@ -99,9 +99,35 @@ void main() async {
   // This also solves the issue with not having Pending (launch) Intents on Linux
   final activeNotification = await taqo_alarm.checkActiveNotification();
   final authState = await GoogleAuth().isAuthenticated;
+
+  // Add licenses manually for Taqo and Alerter. This help to add the licenses
+  // those are not automatically fetched and added by showLicensePage method.
+  loadManualLicenses();
+
   runApp(MyApp(activeNotification: activeNotification, authState: authState));
 }
 
+/// Add the entries to the list to load the particular license manually.
```

Comment:
This is because the comment is not explaining what the function is doing, but instead how to modify the function, or actually what is the list `licenses` is doing.

Reply:
Thank you very much for the comments. I will make the changes.

Reply:
Incorporated the changes.